main.py

```diff
@@ -1,4 +1,13 @@
 import requests 
+from bs4 import BeautifulSoup
 
 players_url =  "https://proclubshead.com/24/club-squad/gen5-179156/" 
-data = requests.get(players_url) 
\ No newline at end of file
+data = requests.get(players_url) 
+
+soup = BeautifulSoup(data.text) 
+members = soup.select('div.bg-gradient')[0] 
+links = members.find_all('a') 
+links = (l.get("href") for l in links) 
```

Comment:
'L' as a variable name isn't that descriptive. Is there a better name for this? e.g `for link in links`